### DISCUSSION TITLE
What is difference between "Talos Cluster" and "Talos Linux Cluster"?

### DISCUSSION THREAD

**Discussion by braveantony** [siderolabs/talos#8007] [General]
I was reading the official Talos document and in it there is a reference to "Talos Cluster" and "Talos Linux Cluster".
What is difference between "Talos Cluster" and "Talos Linux Cluster"?
Can I assume that Talos Cluster represents a group of computers running Talos Linux and Kubernetes at the same time?
Can I assume that Talos Linux Cluster represent a group of computers that are running Talos Linux only?

**Comment by smira** [comment]
There is no difference, it's inconsistent wording. The full name of the operating system is `Talos Linux`, sometimes we use a shorter version `Talos`, but it means same thing - `Talos Linux`.

**Reply by braveantony** [reply]
Thanks for your reply, I understand that Talos is an abbreviation of Talos Linux.
So does Talos Linux Cluster mean a group of computers running Talos Linux and also running Kubernetes? Or just Talos Linux?

**Reply by smira** [reply]
At the moment Kubernetes always runs as part of a Talos Linux machine workload, so there's no difference.

**Reply by braveantony** [reply]
I got it, thanks.
